skada/deep/tests/test_deep_divergence.py

```diff
@@ -1,20 +1,25 @@
 # Author: Theo Gnassounou <theo.gnassounou@inria.fr>
 #         Oleksii Kachaiev <kachayev@gmail.com>
+#         Yanis Lalou <yanis.lalou@polytechnique.edu>
 #
 # License: BSD 3-Clause
 import pytest
 
 pytest.importorskip("torch")
 
 import numpy as np
-import pytest
+import torch
 
 from skada.datasets import make_shifted_datasets
 from skada.deep import DAN, DeepCoral
 from skada.deep.modules import ToyModule2D
 
 
 def test_deepcoral():
+    # Set seed for reproducibility
+    np.random.seed(42)
+    torch.manual_seed(42)
+
```

Comment:
There is the fixture `set_seed` in `conftest.py` that set the seed for every function. Can you add `torch.manual_seed(42)` to this fixture?